homeassistant/components/androidtv/media_player.py

```diff
@@ -192,68 +210,98 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         else:
             device_name = "Android TV / Fire TV device"
 
-        _LOGGER.warning("Could not connect to %s at %s %s", device_name, host, adb_log)
+        _LOGGER.warning(
+            "Could not connect to %s at %s %s", device_name, address, adb_log
+        )
         raise PlatformNotReady
 
-    if host in hass.data[ANDROIDTV_DOMAIN]:
-        _LOGGER.warning("Platform already setup on %s, skipping", host)
+    if address in hass.data[ANDROIDTV_DOMAIN]:
+        _LOGGER.warning("Platform already setup on %s, skipping", address)
+        return
+
+    if aftv.DEVICE_CLASS == DEVICE_ANDROIDTV:
+        device = AndroidTVDevice(
+            aftv,
+            config[CONF_NAME],
+            config[CONF_APPS],
+            config[CONF_GET_SOURCES],
+            config.get(CONF_TURN_ON_COMMAND),
+            config.get(CONF_TURN_OFF_COMMAND),
+        )
+        device_name = config[CONF_NAME] if CONF_NAME in config else "Android TV"
```

Comment:
Use `.get` and pass a default param.
```suggestion
        device_name = config.get(CONF_NAME, "Android TV")
```

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/30086/commits/728b0341c29d2693b5516fcc51cd061843ceca5b